include/freerdp/rail.h

```diff
@@ -419,28 +419,28 @@ struct _RAIL_COMPARTMENT_INFO_ORDER
 	UINT32 ImeState;
 	UINT32 ImeConvMode;
 	UINT32 ImeSentenceMode;
-	UINT32 KanaMode;
+	UINT32 KANAMode;
 };
 typedef struct _RAIL_COMPARTMENT_INFO_ORDER RAIL_COMPARTMENT_INFO_ORDER;
 
-struct _RAIL_ZORDER_SYNC
+struct _RAIL_ZORDER_SYNC_ORDER
 {
 	UINT32 windowIdMarker;
 };
-typedef struct _RAIL_ZORDER_SYNC RAIL_ZORDER_SYNC;
+typedef struct _RAIL_ZORDER_SYNC_ORDER RAIL_ZORDER_SYNC_ORDER;
 
-struct _RAIL_CLOAK
+struct _RAIL_CLOAK_ORDER
 {
 	UINT32 windowId;
-	BOOL cloak;
+	BOOL cloaked;
 };
-typedef struct _RAIL_CLOAK RAIL_CLOAK;
+typedef struct _RAIL_CLOAK_ORDER RAIL_CLOAK_ORDER;
 
-struct _RAIL_POWER_DISPLAY_REQUEST
+struct _RAIL_POWER_DISPLAY_REQUEST_ORDER
 {
-	UINT32 active;
+	BOOL active;
```

Comment:
are you sure? isn't it u32 in docs?

Reply:
It is uint32 in the docs, but the issue is that in EVERY other rail struct that has an integer representing a bool, they have a bool instead of the integer value in the order struct.